packages/commonwealth/client/scripts/views/pages/LaunchToken/steps/TokenInformationStep/TokenInformationForm/TokenInformationForm.tsx

```diff
@@ -0,0 +1,151 @@
+import useAppStatus from 'hooks/useAppStatus';
+import { useBrowserAnalyticsTrack } from 'hooks/useBrowserAnalyticsTrack';
+import React, { useState } from 'react';
+import {
+  BaseMixpanelPayload,
+  MixpanelCommunityCreationEvent,
+  MixpanelLoginPayload,
+} from 'shared/analytics/types';
+import {
+  CWCoverImageUploader,
+  ImageBehavior,
+} from 'views/components/component_kit/cw_cover_image_uploader';
+import { CWLabel } from 'views/components/component_kit/cw_label';
+import { CWTextArea } from 'views/components/component_kit/cw_text_area';
+import { CWButton } from 'views/components/component_kit/new_designs/CWButton';
+import CWCommunitySelector from 'views/components/component_kit/new_designs/CWCommunitySelector';
+import { CWForm } from 'views/components/component_kit/new_designs/CWForm';
+import { CWTextInput } from 'views/components/component_kit/new_designs/CWTextInput';
+import { openConfirmation } from 'views/modals/confirmation_modal';
+import { communityTypeOptions } from 'views/pages/CreateCommunity/steps/CommunityTypeStep/helpers';
+import './TokenInformationForm.scss';
+import { FormSubmitValues, TokenInformationFormProps } from './types';
+import { tokenInformationFormValidationSchema } from './validation';
+
+const TokenInformationForm = ({
+  onSubmit,
+  onCancel,
+}: TokenInformationFormProps) => {
+  const [isProcessingProfileImage, setIsProcessingProfileImage] =
+    useState(false);
+
+  const { isAddedToHomeScreen } = useAppStatus();
+
+  const [baseOption] = communityTypeOptions;
+
+  const { trackAnalytics } = useBrowserAnalyticsTrack<
+    MixpanelLoginPayload | BaseMixpanelPayload
+  >({
+    onAction: true,
+  });
+
+  const handleSubmit = async (values: FormSubmitValues) => {
+    // TODO 8705: call token launch endpoint
+    console.log('values => ', values);
+    onSubmit();
+  };
+
+  const handleCancel = () => {
+    trackAnalytics({
+      event: MixpanelCommunityCreationEvent.CREATE_TOKEN_COMMUNITY_CANCELLED,
+      isPWA: isAddedToHomeScreen,
+    });
```

Comment:
this should be called if "yes" button is clicked I guess

Reply:
ah yes, fixed.